src/main/java/org/broadinstitute/hellbender/tools/exome/HetPulldownCalculator.java

```diff
@@ -98,7 +98,7 @@ public HetPulldownCalculator(final File refFile, final File snpFile,
             }
         }
 
-        Map<Character,Integer> baseCounts = new HashMap<>();
+        final Map<Character, Integer> baseCounts = new HashMap<>();
```

Comment:
hmm yeah, i think i remember you made this change in your liberal pulldown branch that never got merged.  i'll add it back here.  @mbabadi should also take note.